BDPointXamarinBinder.csproj

```diff
@@ -12,12 +12,12 @@
     <OutputType>Library</OutputType>
     <RootNamespace>BDPointXamarinBinder</RootNamespace>
     <AssemblyName>BDPointXamarinBinder</AssemblyName>
-    <TargetFrameworkVersion>v10.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v12.0</TargetFrameworkVersion>
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
     <AndroidClassParser>class-parse</AndroidClassParser>
     <PackageId>Bluedot.PointSDK.Android</PackageId>
-    <PackageVersion>15.4.1</PackageVersion>
+    <PackageVersion>15.5.1.2-alpha</PackageVersion>
```

Comment:
This will be updated to 15.5.1 when min app done QA testing. Ticket has been created [here](https://bluedotinnovation.atlassian.net/browse/BD-4505).

Reply:
You have already uploaded `15.5.1.2-alpha` to nuget.org right? then shouldn't this be just 15.5.1?

Reply:
We haven't published 15.5.1 yet. Doesn't it make more sense to point to the latest version available?

Reply:
Do you plan to upload a new prerelease package after this is merged? If yes then all good. If no then I don't see the point of having it as a prerelease here, since you already generated a QA build.

Reply:
Updated.